src/extlibs/enlsvg/Pathfinding/ENLSVGGraph.h

```diff
@@ -17,13 +17,13 @@ namespace Pathfinding
     {
         struct EdgeData
         {
-            const VertexID sourceVertex;
-            const VertexID destVertex;
+            const VertexID source_vertex;
+            const VertexID dest_vertex;
             int level;
-            std::vector<EdgeID> tautOutgoingEdges;
+            std::vector<EdgeID> taut_outgoing_edges;
 
             EdgeData(VertexID sourceVertex, VertexID destVertex, int level)
-                : sourceVertex(sourceVertex), destVertex(destVertex), level(level)
+                : source_vertex(sourceVertex), dest_vertex(destVertex), level(level)
```

Comment:
`sourceVertex` and `destVertex` should be updated, similar with multiple other function parameters in this file (e.g. `imNext` and `imLast`)

Reply:
done